data/strings/types_strings.txt

```diff
@@ -14609,6 +14609,46 @@
     zh-Hans = 高塔
     zh-Hant = 塔
 
+  [type.man_made.tower.communication]
+    en = Cell Tower
+    ar = ﺕﻻ﻿ﺎﺼﺗﻻ﻿ﺍ ﺝﺮﺑ
```

Comment:
Arabic, Hebrew and Persian are garbaged now by the translation script. Or it could be a text direction issue.
```suggestion
    ar = برج الخلوي
```

Reply:
See #4522